<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerPage</class>
 <widget class="QWidget" name="ManagerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ERROR: The Python engine could not be initialised!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <widget class="QWidget" name="manageTab">
      <attribute name="title">
       <string>Manager</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reloading a running plugin may have unpredictable effects. Consider saving your work before using this command.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KPushButton" name="reload">
         <property name="text">
          <string>Reload Plugins</string>
         </property>
         <property name="icon">
          <iconset theme="system-reboot">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTreeView" name="tree"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
